source/config-settings/config-file-options.txt

```diff
@@ -0,0 +1,63 @@
+.. _config-file-options:
+
+-----------------------------------
+Compass Configuration File Settings
+-----------------------------------
+
+.. default-domain:: mongodb
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 1
+   :class: singlecol
+
+You can configure |compass| using a configuration file. The configuration file 
+configures settings at startup. Settings configured using the configuration file 
+cannot be modified on the |compass-short| interface. For more detailed guidance 
+on how to configure a setting, see the task page for the specific setting.
+
+Syntax
+------
+
+You can configure |compass-short| configuration file settings in either EJSON or 
+YAML format.
+
+EJSON
+`````
+
+.. code-block:: json
+
+    { 
+      "setting": value,
+      "setting": value
```

Comment:
[nit]

Maybe add ellipses after these rows to indicate that you can specify many options?